src/Mod/Fem/App/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ else(MSVC)
 endif(MSVC)
 
 if(BUILD_FEM_NETGEN)
-    add_definitions(-DFCWithNetgen)
+    add_definitions(-DFCWithNetgen ${NETGEN_DEFINITIONS})
```

Comment:
This isn't currently necessary, but I thought it would be a good idea in case we add something new to NETGEN_DEFINITIONS (and would expect those to apply where nglib is used).